doc/akri_connector/connector-metadata-schema.json

```diff
@@ -0,0 +1,337 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "aioConnectorMetadataSchemaVersion": "1.0",
+  "definitions": {
+    "schema": {
+      "type": "object",
+      "description": "The schema for a stringified JSON schema fields such as 'additionalConfigurationSchema'. Must provide inline schema xor the file that contains the schema. These schemas may be used to define the default values of unspecified fields.",
+      "properties": {
+        "schema": {
+          "$ref": "http://json-schema.org/draft-07/schema#"
+        },
+        "schemaFile": {
+          "type": "string",
+          "description": "A path to another file that contains the schema. Typically, both the connector metadata file and its schema files live in the same container registry alongside the connector image."
+        }
+      },
+      "oneOf": [
+        {
+          "required": [
+            "schema"
+          ]
+        },
+        {
+          "required": [
+            "schemaFile"
+          ]
+        }
```

Comment:
I would always go with inline schema, then we have one artifact to read whiich sits in the CR. Specifying a schema file could cause issues re: connectivity and authorization, which end up in IcM.

Reply:
@varunpuranik suggested allowing file refs elsewhere in this PR because he expects some schemas to be very large. I'd personally rather schemas always be inline as well, though. @varunpuranik  thoughts?

Reply:
I talked with Varun offline and he is fine with forcing inline here. I'll make the change